prescription-sync-job/base/cronjob.yaml

```
@@ -0,0 +1,114 @@
+---
+kind: CronJob
+apiVersion: batch/v1beta1
+metadata:
+  name: prescription-sync
+  labels:
+    app: thoth
+    component: prescription-sync
+spec:
+  schedule: "@daily"
+  suspend: true
+  concurrencyPolicy: Forbid
+  successfulJobsHistoryLimit: 2
+  failedJobsHistoryLimit: 4
```

```suggestion
  successfulJobsHistoryLimit: 1
  failedJobsHistoryLimit: 2
```
I think 2 failed jobs history is good enough.